packages/lib/OgImages.tsx

```diff
@@ -139,7 +139,7 @@ export const Meeting = ({ title, users = [], profile }: MeetingImageProps) => {
       <div tw="h-full flex flex-col justify-start">
         <div tw="flex items-center justify-center" style={{ fontFamily: "cal", fontWeight: 300 }}>
           <img src={`${CAL_URL}/${LOGO}`} width="350" alt="Logo" />
-          {avatars.length > 0 && <div tw="font-bold text-emphasis text-[92px] mx-8 bottom-2">/</div>}
```

Comment:
@zomars other lines in the file also use  emphasis. should we remove them all ?
we can maybe also use  style=  {{ color: "var("--cal-text-emphasis")" }}